L.Map.Sync.js

```diff
@@ -17,7 +17,7 @@
     // with the top right corner of the target map.
     // The values can be less than 0 or greater than 1. It will sync
     // points out of the map.
-    L.Util.offsetHelper = function (ratioRef, ratioTgt) {
```

Comment:
Just adding a function to L.Util does not work in my Ubuntu Chrome (I have not tested in other browsers). Any idea how to do it? The only option I see is to include it directly in `L`, not in `L.Util`, but it breaks backwards compatibility 😞